Source/cmake/OptionsWPE.cmake

```diff
@@ -11,6 +11,8 @@ endif ()
 
 set(USER_AGENT_BRANDING "" CACHE STRING "Branding to add to user agent string")
 
+find_package(ATK 2.16.0)
+find_package(ATKBridge)
```

Comment:
```suggestion
find_package(ATK 2.16.0 REQUIRED)
find_package(ATKBridge REQUIRED)
```

Reply:
(We can remove them later when adding `USE(ATK)`, but for now, they are required.)